examples/elm-architecture-tutorial/Example1/Counter.elm

```diff
@@ -56,20 +57,19 @@ view : Model -> Html Action
 view model =
     div []
         [ button [ onClick Decrement ] [ text "-" ]
-        , div [ countStyle ] [ text (toString model) ]
+        , div countStyle [ text (String.fromInt model) ]
         , button [ onClick Increment ] [ text "+" ]
         ]
 
 
-countStyle : Attribute any
+countStyle : List (Attribute any)
```

Comment:
I know this is in _the original_, but I really like the use of `any` as the name of a type variable.